### ISSUE TITLE
 Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown

### ISSUE THREAD

**Issue by fusion809** [MustaphaTR/Romanovs-Vengeance#27]
Executing the AppImage belonging to commit db524ef574688981d904e95a723dd60090a20923, returns the exception:

```bash
Loading mod: rv
Exception of type `System.Reflection.ReflectionTypeLoadException`: Exception of type 'System.Reflection.ReflectionTypeLoadException' was thrown.
Could not resolve type with token 0100010f from typeref (expected class 'OpenRA.Mods.Common.Traits.IResourceLogicLayer' in assembly 'OpenRA.Mods.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null')
  at (wrapper managed-to-native) System.Reflection.Assembly.GetTypes(System.Reflection.Assembly,bool)
  at System.Reflection.Assembly.GetTypes () [0x00000] in <6649516e5b3542319fb262b421af0adb>:0 
  at OpenRA.Exts.GetNamespaces (System.Reflection.Assembly a) [0x00000] in <b732385ffe7749859267ee913842063c>:0 
  at OpenRA.ObjectCreator+<>c.<.ctor>b__4_0 (System.Reflection.Assembly asm) [0x0000d] in <b732385ffe7749859267ee913842063c>:0 
  at System.Linq.Enumerable+SelectManySingleSelectorIterator`2[TSource,TResult].ToArray () [0x00028] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at System.Linq.Enumerable.ToArray[TSource] (System.Collections.Generic.IEnumerable`1[T] source) [0x0001f] in <35ad2ebb203f4577b22a9d30eca3ec1f>:0 
  at OpenRA.ObjectCreator..ctor (OpenRA.Manifest manifest, OpenRA.InstalledMods mods) [0x0010e] in <b732385ffe7749859267ee913842063c>:0 
  at OpenRA.ModData..ctor (OpenRA.Manifest mod, OpenRA.InstalledMods mods, System.Boolean useLoadScreen) [0x00029] in <b732385ffe7749859267ee913842063c>:0 
  at OpenRA.Game.InitializeMod (System.String mod, OpenRA.Arguments args) [0x0015a] in <b732385ffe7749859267ee913842063c>:0 
  at OpenRA.Game.Initialize (OpenRA.Arguments args) [0x00468] in <b732385ffe7749859267ee913842063c>:0 
  at OpenRA.Game.InitializeAndRun (System.String[] args) [0x00006] in <b732385ffe7749859267ee913842063c>:0 
  at OpenRA.Program.Main (System.String[] args) [0x00044] in <b732385ffe7749859267ee913842063c>:0 

AL lib: FreeContext: (0x1671000) Deleting 32 Source(s)
```

Full logs are here: [logs-20190625.zip](https://github.com/MustaphaTR/Romanovs-Vengeance/files/3322078/logs-20190625.zip).

**Comment by GraionDilach** [comment]
Yes, this is still an issue. I pointed it out on #1's name. As long as the AS and the RV namespace has traits named the same, this remains.

**Comment by fusion809** [comment]
Does appear fixed as of commit 1811 (648f5e519940c9d66ea6a1deb84cd023fa9a092b).
